scipy/odr/odrpack.py

```diff
@@ -171,9 +181,10 @@ class Data(object):
     Parameters
     ----------
     x : array_like
-        Input data for regression.
-    y : array_like, optional
-        Input data for regression.
+        Observed data for the independent variable of the regression
+    y : array_like or scalar, optional
```

Comment:
About the "implicit model" - this raised the question for me whether the behavior for scalar and single-element arrays is the same or not. Is it?